include/ggml/ggml.h

```diff
@@ -531,36 +533,35 @@ extern "C" {
         int n_threads;
 
         // the `n_tasks` of nodes, 1:1 mapping to cgraph nodes
-        int n_tasks[GGML_MAX_NODES];
+        int n_tasks[GGML_DEFAULT_GRAPH_SIZE]; // FIXME: need to allocate ggml_cplan dynamically?
```

Comment:
The array of `n_tasks` in `ggml_cplan` would need to be dynamically allocated to account for varying graph sizes. Maybe we could just remove it and replace it with a lookup table per OP?

Reply:
The unary ops might be problematic because some of them are parallel and some of the are not.

Can we remove `n_tasks` all together as a member, and instead convert it to a function:

```c
int ggml_cplan_n_tasks(struct ggml_tensor * node);
```

Don't think we need a storage for `n_tasks` - can be computed on the fly?